### DISCUSSION TITLE
Incredible Work

### DISCUSSION THREAD

**Discussion by kran27** [kotyaralih/NostalgiaCore#36] [General]
just wanted to thank you, I wanted to move my 0.8.1 survival world onto a server so i dont have to transfer between devices, this software actually works properly (unlike PocketMine 1.3.12).

**Comment by GameHerobrine** [comment]
*well i wouldnt say that all the vanilla features work properly*

**Reply by kran27** [reply]
every single issue i noticed in the time i used pocketmine has worked essentially perfectly here. the issues i have had didn't result in me losing and progress or dying, unlike pocketmine.
